### DISCUSSION TITLE
Please add GCC/G++ 7 as the Required Packages in the WiKi page "Building OpenRCT2 on Linux"

### DISCUSSION THREAD

**Discussion by hyh19962008** [OpenRCT2/OpenRCT2#18434] [Feature requests]
Currently the the code used C++17 features such as std::optional  and std::string_view, which only supported with GCC 7 and above. I get compilation errors of missing header files like <optional> and <string_view> when trying to compiler the code on Ubuntu 16.04 with GCC 5 or GCC 6.

**Comment by janisozaur** [comment]
You are welcome to add that info yourself

**Reply by hyh19962008** [reply]
OK, will do.
